docs/users-guide/04-asking-questions.md

```diff
@@ -1,85 +1,115 @@
 
-Metabase's two core concepts are questions and their corresponding answers. Everything else is based around questions and answers. To ask a question in Metabase, click the Ask a Question button at the top of the screen.
+Metabase's two core concepts are questions and their corresponding answers. Everything else is based around questions and answers. To ask a question in Metabase, click the **+ New** button in the upper right of the main navigation bar, and select either:
 
-### Ways to start a new question
+- Question
+- [SQL query](./writing-sql.md)
 
-![The new question menu](./images/new-question.png)
+This page covers how to ask a question using Metabase's graphical query builder, the "Question" option.
 
-When you click the Ask a Question button, you'll see that there are three ways to ask a specific question in Metabase:
+## Creating a new question with the query builder
 
-1. The simple question mode, which lets you filter, summarize, and visualize data.
-2. [The custom question mode](custom-questions.md), which gives you a powerful notebook-style editor to create more complex questions that require joins, multiple stages of filtering and aggregating, or custom columns.
-3. [The SQL/native query editor](writing-sql.md).
+From the **+ New** dropdown, select **Question**, then pick your starting data:
 
-This page will cover the simple mode.
+You can start a question from:
 
-## Asking a simple question
+- **A model**. A [model][model] is a special kind of saved question meant to be used as a good starting point for questions. Sometimes these are called derived tables, as they usually pull together data from multiple raw tables.
+- **Raw data**. You'll need to specify the database and the table in that database as the starting point for your question.
+- A **saved question**. You can use the results of any question as the starting point for a new question.
 
-After you select the Simple Question option, you'll need to pick some data that you have a question about.
+Note that there are some kinds of saved questions that can't be used as source data:
 
-### Picking data
+- Druid questions
+- Google Analytics questions
+- Mongo questions
+- Questions that use `Cumulative Sum` or `Cumulative Count` aggregations
+- Questions that have columns that are named the same or similar thing, like `Count` and `Count 2`
 
-Just pick one of the databases that's connected to Metabase, and you'll see a list of all the tables inside of it. Pick the one that you have a question about; it could be a Users table, or maybe it's something like Events, Orders, or Downloads.
+## The query builder
 
-Metabase doesn't support joining tables _across_ multiple databases, and for good reason: it's generally better to bring related data into a single database.
+Once you select your data, Metabase will take you to the notebook editor of the query builder. Say you selected **Raw data** > **Sample Dataset** > **Orders**, then you'll see something like this:
```

Comment:
> the notebook editor of the query builder

This feels awkward. I wonder if we need the term "notebook editor" anymore, or if it's just "the query builder" now that it's the default pace you're dropped in when doing a GUI query.